py/server/deephaven/constants.py

```diff
@@ -9,7 +9,7 @@
 _JQueryConstants = jpy.get_type("io.deephaven.util.QueryConstants")
 
 # Deephaven Special Null values for primitive types
-NULL_BOOLEAN = _JQueryConstants.NULL_BOOLEAN
+NULL_BOOLEAN = _JQueryConstants.NULL_BYTE
```

Comment:
Is this change intentional?  If so, it seems to deserve a comment.  This makes me nervous without justification.

Reply:
yes, left a comment